### PR TITLE
Improve wording in branching policy

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: news/5372.documentation [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve wording in branching policy @sneridagh
```
Comment (stevepiercy) [inline news/5372.documentation:1]:
```suggestion
Improved wording in branching policy. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Use past tense on news item.

File: docs/source/contributing/branch-policy.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ canary
 `17.x.x`
 :   `17.x.x` is the current stable and actively developed branch in git.
     Upon the final release of version `18.0.0`, the `17.x.x` branch line will be no longer supported.
-    Any new feature will be generally released to the `main` branch, and only backported to older versions if the Volto Team approves it.
+    Any new feature will be merged into the `main` branch, and only backported to and released in older versions, if the Volto Team approves it.
```
Comment (davisagli) [inline docs/source/contributing/branch-policy.md:35]: And the same change on line 40, please

Comment (sneridagh) [reply docs/source/contributing/branch-policy.md:35]: Done.

**Review by stevepiercy — APPROVED** [review]
Good catch @davisagli

**Pull request merged**